spec/service_models/miq_ae_service_ems_event_spec.rb

```diff
@@ -1,5 +1,7 @@
 describe MiqAeMethodService::MiqAeServiceEmsEvent do
   before do
+    Settings.ems_refresh.vmwarews.streaming_refresh = false
```

Comment:
Is this going to cause cross-test corruption.  Does this value need to be reset to the initial value in an `after` block?

Reply:
`stub_settings_merge(:ems_refresh => {:vmwarews => {:streaming_refresh => false}})` would be the better way to do this, but it won't cause test contamination afaik